tests/src/JIT/Regression/JitBlue/DevDiv_284785/DevDiv_284785.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
1. It is useful to have a comment stating what is intended to be tested, e.g., what failed before and why.
2. I think the standard failure result code is 101, not zero.